docs/en/operation/Basic-Logging.md

```diff
@@ -123,10 +114,10 @@ Furthermore, you can turn on Alluxio debug logging when you are troubleshooting
 in a running cluster, and turn it off when you are done.
 ```console
 # Turn on Alluxio debug logging and start debugging
-$ ./bin/alluxio logLevel --logName=alluxio --target=master,workers --level=DEBUG
```

Comment:
Now the valid targets is no longer just `master,workers`. By not specifying the target, all master/job_master/workers/job_workers are updated.